ui/tslint.json

```diff
@@ -25,13 +26,19 @@
     "label-position": true,
     "max-line-length": [
       true,
-      100
+      140
```

Comment:
I'll set it back down (this came in as a default among a bunch of settings from the output of `ng new`).